### PR TITLE
Feature/fad 6081

### PR DESCRIPTION
**Pull request by pmcclory**

### CODE REVIEW

File: lib/dynamo.js [hunk 1]
```diff
@@ -22,8 +22,20 @@ function createClient(config) {
...
   const client = new AWS.DynamoDB.DocumentClient(config);
+  client.consistentReads = 'consistentReads' in config && config.consistentReads;
   Promise.promisifyAll(client);
 
+  // patch read methods to use consistent read if specified
+  _.forEach(['queryAsync', 'getAsync', 'scanAsync'], function(fname) {
+    var tmpFunc = client[fname];
+    client[fname] = function wrapper(params) {
+      if (this.consistentReads && (fname != 'queryAsync' || !('IndexName' in params))) {
+        params.ConsistentRead = true;
```
Comment (bizob2828) [inline lib/dynamo.js]: do we have to pay more for this?

File: lib/dynamo.js [hunk 2]
```diff
@@ -22,8 +22,20 @@ function createClient(config) {
   }
 
   const client = new AWS.DynamoDB.DocumentClient(config);
+  client.consistentReads = 'consistentReads' in config && config.consistentReads;
```
Comment (bizob2828) [inline lib/dynamo.js]: say what now? what are you trying to do here? make sure it's not undefined? you can use `_.get(config, 'consistentReads', false)`

Comment (pmcclory) [reply lib/dynamo.js]: sure, good call

File: lib/dynamo.js (hunk 1 above)
Comment (pmcclory) [reply lib/dynamo.js]: Not directly, but it does consume more RCU.  We're only talking about turning this on in for tests though, not across the board, see:

https://github.com/SparkPost/accusers-api/pull/288

File: lib/dynamo.js (hunk 2 above)
Comment (bizob2828) [reply lib/dynamo.js]: or just config.consistenReads || false since i doubt config will ever be undefined

**Pull request closed without merging**